src/main/java/com/revature/repositories/BatchRepository.java

```diff
@@ -0,0 +1,25 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import com.revature.domain.Batch;
+
+@Repository
+public interface BatchRepository extends JpaRepository<Batch, Integer> {
+	
+	Batch findBatchByName(String name);
+	Batch findBatchById(int id);
+	void delete(Batch batchName);
+	Batch save(Batch batchName);
+	List<Batch> findAll();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1161)